sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -102,7 +102,7 @@ def format_mapping_identifier(string):
 
 
 def logical_port_name_to_physical_port_list(port_name):
-    if port_name.startswith("Ethernet"):
+    if re.search(swsscommon.FRONT_PANEL_PORT_PREFIX_REGEX, port_name):
```

Comment:
Use `re.match` to match the whole `port_name`.

Reply:
done